src/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 'stable'
+cache:
+  directories:
+    - node_modules
```

Comment:
I've had some issues with invalid caches in the past, but this will speed up the runs so lets just keep an eye on this.